build_runner/lib/src/generate/build_impl.dart

```diff
@@ -50,15 +51,17 @@ Future<BuildResult> build(List<BuildAction> buildActions,
     bool skipBuildScriptCheck,
     bool enableLowResourcesMode}) async {
   var options = new BuildOptions(
-      deleteFilesByDefault: deleteFilesByDefault,
-      writeToCache: writeToCache,
+      deleteFilesByDefault: deleteFilesByDefault ?? writeToCache,
```

Comment:
I would add an extra todo here or something to make sure we clean it up.

Just so I understand correctly - setting `writeToCache: true` effectively just makes all your build actions hidden (maintains old behavior?)

Reply:
yes, the setting maintains behavior. I think this is covered by the TODO to remove the argument.